redshift_sqlalchemy/dialect.py

```diff
@@ -12,6 +12,9 @@
 except ImportError:
     pass
 else:
+    from alembic.ddl.base import RenameTable
+    compiles(RenameTable, 'redshift')(postgresql.visit_rename_table)
+
     class RedshiftImpl(postgresql.PostgresqlImpl):
```

Comment:
well Redshift should have an alembic impl called RedshiftImpl and use `__dialect__ = 'redshift'`, subclassing PostgresqlImpl